pkg/api/v1/conversion.go

```diff
@@ -627,6 +638,11 @@ func Convert_v1_Pod_To_api_Pod(in *Pod, out *api.Pod, s conversion.Scope) error
 		// back to the caller.
 		in.Spec.InitContainers = values
 	}
+	// If there is a beta annotation, copy to alpha key.
+	// See commit log for PR #31026 for why we do this.
+	if valueBeta, okBeta := in.Annotations[PodInitContainerStatusesBetaAnnotationKey]; okBeta {
```

Comment:
We do this a couple lins above (line 623) are we doing this twice intentionally?